test/remote/gateways/remote_reach_test.rb

```diff
@@ -74,6 +75,60 @@ def test_failed_capture
     assert_equal 'Not Found', response.message
   end
 
+  #
+  # The following refund test was commented or it is optional because take
+  # ~1 minute to get a successful response
+  #
+
+  # def test_successful_refund
+  #   purchase = @gateway.purchase(@amount, @credit_card, @options)
+  #   sleep(1.minute)
+  #   response = @gateway.refund(@amount, purchase.authorization, { reference_id: 'REFUND_TAG' })
+
+  #   assert_success response
+  #   assert response.params['response'][:RefundId].present?
+  # end
```

Comment:
💬  Comment:

If refund responses are idempotent isn't possible to have a test with fixed orderId value an use that just to run the refund code ans ensure that works properly ?

Reply:
Done!